core/templates/domain/exploration/SolutionObjectFactory.ts

```diff
@@ -64,7 +67,7 @@ export interface ShortAnswerResponse {
 
 export class Solution {
   ehfs: ExplorationHtmlFormatterService;
-  shof: SubtitledHtml;
+  shof!: SubtitledHtml;
```

Comment:
Is non-null assertion required here? If not lets remove it. If required, add a comment like added at other places in the codebase

Reply:
This property is not used / initialised anywhere in the file, not in the spec file too. Should I remove the property itself?

Reply:
Yep, can remove this

Reply:
done